packages/SwingSet/src/runPolicies.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+
+import { assert } from '@agoric/assert';
+
+export function foreverPolicy() {
+  /** @type { RunPolicy } */
+  return harden({
+    vatCreated(_details) {
+      return true;
+    },
+    crankComplete(_details) {
+      return true;
+    },
+    crankFailed(_details) {
+      return true;
+    },
+  });
+}
+
+export function crankCounter(maxCranks, maxCreateVats) {
```

Comment:
Seems like there's a role for something that counts cranks but really doesn't care about vat creations.  I suppose you could get that by passing `Number.MAX_INTEGER_VALUE` for `maxCreateVats`, but I'm left wondering if there should be an explicit "don't care" signal, like 0 or -1 or something.

Reply:
Eh, `runPolicies.js` is more about providing examples than trying to cover the full range of what sort of policy you might want to implement. Mostly I wanted something to establish the intuition that creating a whole new vat is tremendously more expensive than sending a message to an existing one.